rasa_core/channels/channel.py

```diff
@@ -67,10 +67,35 @@ def register(input_channels,  # type: List[InputChannel]
 
 def button_to_string(button, idx=0):
     """Create a string representation of a button."""
-    return "{idx}: {title} ({val})".format(
-            idx=idx + 1,
-            title=button.get('title', ''),
-            val=button.get('payload', ''))
+
+    title = button.pop('title', '')
+    payload = button.pop('payload', '')
+    if payload:
+        payload = "({})".format(payload)
+
+    button_string = "{idx}: {title} {payload}".format(
+                     idx=idx + 1,
+                     title=title,
+                     payload=payload)
+
+    if button:
```

Comment:
logically, I'd go for the same code style for how `payload ` and `button` are handled. how about this:
```python
    if 'payload' in button:
        payload = " ({})".format(button.pop('payload'))
    else:
        payload = ""
     
    # if there are any additional attributes, we append them to the output
    if button:
        details = " - {}".format(json.dumps(button, sort_keys=True))
    else:
        details = ""

    button_string = "{idx}: {title}{payload}{details}".format(
                     idx=idx + 1,
                     title=title,
                     payload=payload,
                     details=details)
```